extensions/apps/antenna/src/components/pages/global-antenna-page/index.tsx

```diff
@@ -60,6 +60,11 @@ const GlobalAntennaPage: React.FC<unknown> = () => {
     <Stack className="w-full">
       <Helmet helmetData={helmetData}>
         <title>{worldConfig.title}</title>
+        <style>{`
+          [data-scroll-locked][data-scroll-locked] {
```

Comment:
i'm not happy with this approach:
1. it overflows the context of an app - other apps will get this style without needing it
2. it is hard to track which app is responsible of adding this style
are you using a library that doesn't have an option to style this particular state?

Reply:
the radix ui dropdown menu implementation removes the window scroll bar when the drop down is active which from the doc I didn't find a way to solve it that I went with introducing that style to global antennna

but now i went through their github and  figured i can turn off the modal prop of the drop down menu which helps to prevent the logic that removes the window scroll bar but on the other side now when the modal is active one can interact with other things beside the modal which assessed is fine